### ISSUE TITLE
Add version buttons to website

### ISSUE THREAD

**Issue by redeboer** [ComPWA/pycompwa#73]
Since we now host [our website](https://compwa.github.io/) through GitHub Pages and not through [Read the Docs](https://readthedocs.org/), we do not have versioning of the website anymore. This is a problem if you want to view the API of older versions.

Possible solutions:
* [`sphinxcontrib-versioning`](https://sphinxcontrib-versioning.readthedocs.io/) (seems to be better maintained)
* [`sphinx-versions`](https://sphinx-versions.readthedocs.io/)

**Comment by spflueger** [comment]
Hmm, since we probably wont have multiple versions I think this is not necessary or low priority

**Comment by redeboer** [comment]
True, it won't be necessary for `pycompwa`, but this is a general issue that can be transferred to [`tensorwaves`](https://github.com/ComPWA/tensorwaves) once relevant.

**Comment by spflueger** [comment]
yeah, but there we could actually consider going back to read the docs

**Comment by redeboer** [comment]
Yes, but we switched to GitHub Pages to standardise our urls:
* [compwa.github.io](https://compwa.github.io/) for pycompwa
* [compwa.github.io/ComPWA](https://compwa.github.io/) for the C++ backend

In this way, you can unify the urls of several packages:
* `compwa.github.io` (e.g. overview and theory)
* `compwa.github.io/tensorwaves`
* `compwa.github.io/ComPWA`
* `compwa.github.io/pycompwa`
